google/cloud/pubsub_v1/subscriber/scheduler.py

```diff
@@ -162,7 +162,7 @@ def shutdown(
                 work_item = self._executor._work_queue.get(block=False)
                 if work_item is None:  # Exceutor in shutdown mode.
                     continue
-                dropped_messages.append(work_item.args[0])
+                dropped_messages.append(work_item.args[0])  # type: ignore[index]
```

Comment:
Can you add a comment to provide more detail on why we are ignoring the type error? If there is follow up work, please create a new issue and link to it here.